lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/table/TableRowTruthDAOImpl.java

```diff
@@ -67,7 +67,7 @@
  * @author John
  * 
  */
-public class TableRowTruthDAOImpl implements TableRowTruthDAO {
+public abstract class TableRowTruthDAOImpl implements TableRowTruthDAO {
```

Comment:
I would say that instead of making this abstract and having TableRowTruthDAOImpl no longer be independent of CachingTableRowTruthDAOImpl, checkForRowLevelConflict should probably be pushed down from CachingTableRowTruthDAOImpl to TableRowTruthDAOImpl, it doesn't belong in the caching versions anymore.